test/com/google/enterprise/adaptor/database/ResponseGeneratorTest.java

```diff
@@ -247,14 +250,13 @@ public void testRowToText_specialCharacters() throws Exception {
     Response response = newProxyInstance(Response.class, bar);
     Map<String, String> cfg = new TreeMap<String, String>();
     ResponseGenerator resgen = ResponseGenerator.rowToText(cfg);
-
     ResultSet rs = executeQueryAndNext("select * from data");
```

Comment:
Add a blank line before this (after resgen declaration). I think I had you remove it in a patch set, but it was there in the original.